yamlReader.py

```diff
@@ -0,0 +1,167 @@
+import yaml
+import yamlordereddictloader
+import start
+
+yamlName = start.yamlName()
+
+
+def getDefault():
+    defaultDay = -1
+    with open(yamlName, "r") as f:
+        # yaml file opening and loading
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            # fetching employees list
+            if (attribute == "default_date_day"):
+                defaultDay = int(value)
+    return defaultDay
+
+
+def getEmployeesList():
+    with open(yamlName, "r") as f:
+        # yaml file opening and loading
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            # fetching employees list
+            if (attribute == "employees"):
+                employeesList = []
+                for employee in value:
+                    for (attribute, value) in employee.items():
+                        if (attribute == "employee"):
+                            employeesList.append(value)
+    return employeesList
```

Comment:
You are at the risk to return something undefined here

Reply:
Changed so it returns an empty list if it doesn't find any employees, same for getRulesList